{
	"header": {
		"line": "CVE-2022-48521 (An issue was discovered in OpenDKIM through 2.10.3, and 2.11.x through ...)",
		"name": "CVE-2022-48521",
		"description": "(An issue was discovered in OpenDKIM through 2.10.3, and 2.11.x through ...)"
	},
	"annotations": [
		{
			"line": "\t{DLA-3680-1}",
			"type": "xref",
			"bugs": [
				"DLA-3680-1"
			]
		},
		{
			"line": "\t- opendkim 2.11.0~beta2-9 (bug #1041107)",
			"type": "package",
			"package": "opendkim",
			"kind": "fixed",
			"version": "2.11.0~beta2-9",
			"flags": [
				{
					"bug": 1041107
				}
			]
		},
		{
			"line": "\t[bookworm] - opendkim 2.11.0~beta2-8+deb12u1",
			"type": "package",
			"release": "bookworm",
			"package": "opendkim",
			"kind": "fixed",
			"version": "2.11.0~beta2-8+deb12u1"
		},
		{
			"line": "\t[bullseye] - opendkim 2.11.0~beta2-4+deb11u1",
			"type": "package",
			"release": "bullseye",
			"package": "opendkim",
			"kind": "fixed",
			"version": "2.11.0~beta2-4+deb11u1"
		},
		{
			"line": "\tNOTE: https://github.com/trusteddomainproject/OpenDKIM/issues/148",
			"type": "NOTE",
			"description": "https://github.com/trusteddomainproject/OpenDKIM/issues/148"
		}
	]
}
